Makefile

```diff
@@ -58,7 +58,9 @@ build-amd64:
 
 .PHONY: build-386
 build-386:
+ifneq ($(shell uname -s), Darwin)
```

Comment:
Does this solve the issue you mentioned on slack: https://kubernetes.slack.com/archives/C013FQNB0A2/p1610469540144000 ?

Reply:
No, it does not. I was trying to see if I could get it to work. It still doesn't

Reply:
Thank you for the update, which GOOS and GOARCH combination are you using?